SMARTeacher/mockgraphql
# Add environment variable to know if we want to use fixture or not

## Description
This PR is to add an environment variable which determines whether we want to use the fixtures or use random values in our resolvers.

## How should this be manually tested?
`FIXTURE=true npm run start` -> Variable should be true in the code
![image](https://user-images.githubusercontent.com/49197386/62805486-22afa600-babe-11e9-9112-3c078be9cf54.png)


`npm run start` -> Variable should be false
![image](https://user-images.githubusercontent.com/49197386/62805462-1297c680-babe-11e9-98d5-0a9732e6666d.png)
